modules/ztp-deploying-user-defined-firmware-configuration-with-gitops-ztp.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-deploying-user-defined-firmware-configuration-with-gitops-ztp_{context}"]
+= Deploying user-defined firmware to cluster hosts with {ztp}
+
+You can deploy user-defined firmware to cluster hosts by configuring the `SiteConfig` custom resource (CR) to include a hardware profile that you want to apply during cluster host provisioning.
+You can configure hardware profiles to apply to cluster hosts in the following scenarios:
+
+* All hosts in the cluster
+* Only hosts that meet certain criteria in the cluster
+* Individual hosts in the cluster
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed {rh-rhacm-first} and logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have provisioned a cluster that is managed by {rh-rhacm}.
+
+* You created a Git repository where you manage your custom site configuration data.
+The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+.Procedure
+
+. Create the host firmware profile files that contain the firmware settings you want to apply at the site, cluster, and individual node level.
+For example:
++
+.site-firmware.profile
+[source,yaml]
+----
+BootMode: Uefi
+LogicalProc: Enabled
+ProcVirtualization: Enabled
+----
++
+.cluster-firmware.profile
+[source,yaml]
+----
+EnergyEfficientTurbo: Enabled
+ProcC1E: Disabled
+TpmSecurity: "On"
+----
++
+.node-firmware.profile
+[source,yaml]
+----
+EnergyEfficientTurbo: Disabled
+ProcTurboMode: Enabled
+ProcPwrPerf: OsDbpm
+ProcC1E: Disabled
+ProcCStates: Enabled
```

Comment:
These profile examples from the QE Git repository are designed to test the hierarchy of profiles, where the node-level profile overrides the cluster-level profile, and the cluster-level profile overrides the site-level profile.
For the custom documentation, we only need one profile example. This example should demonstrate how to apply the profile at the site level, cluster level, or node level, while noting that the node-level profile will overwrite both the cluster-level and site-level profiles.